controllers/datafoundation.go

```diff
@@ -221,3 +231,25 @@ func (r *dataFoundationReconciler) reconcileCephIngressNetworkPolicy() error {
 	}
 	return nil
 }
+
+func (r *dataFoundationReconciler) reconcileRookCephOperatorConfig() error {
+	if err := r.get(r.rookConfigMap); err != nil {
+		// Because resource limits will not be set, failure to get the Rook ConfigMap results in failure to reconcile.
```

Comment:
This is an error about resource limits, but I don't see any code that update the resource limits